lib/md009.js

```diff
@@ -30,9 +30,15 @@ module.exports = {
       listItemLineNumbers.sort((a, b) => a - b);
     }
     const expected = (brSpaces < 2) ? 0 : brSpaces;
-    forEachLine(lineMetadata(), (line, lineIndex) => {
+    let inFencedCode = false;
+    forEachLine(lineMetadata(), (line, lineIndex, inCode, onFence) => {
+      if (onFence > 0) {
```

Comment:
Sorry, I thought this would be easier.

How about:
inFencedCode=0
And:
inFencedCode+=onFence

Shorter and handles nesting?